src/Homework1.java

```diff
@@ -1,101 +1,224 @@
 public class Homework1 {
 
     public static void main(String[] args) {
-        // Primitives
-        int myPrimitive = 10;
-        System.out.println(Integer.toBinaryString(myPrimitive));
-        System.out.println(Integer.toBinaryString(Integer.MAX_VALUE));
+        // 1. Write a program that will sum up all numbers from 0 up to a target number.
+        int[] ints = {0, 1, 2, 3, 4, 5};
+        int sum = 0;
+        for (int i = 0; i <= 5; i++) {
+            sum = sum + ints[i];
+        }
+        System.out.println(sum);
+
+
+        // 2. Write a program that will calculate the average and sum of all numbers between the range of 2 numbers.
+        int numbers = 100;
+        int sum1 = 1;
+        int average = 1;
+        //sum
+        for (int j = 0; j <= numbers; j += 2) {
+            sum1 = sum1 + j;
```

Comment:
Asta o as iti faca suma a jumatate din numere (0,2,4,6,8,10...) de ce faci pasul din 2 in 2?